images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -70,6 +70,7 @@ $markdown += New-MDList -Style Unordered -Lines (@(
 $markdown += New-MDHeader "Tools" -Level 3
 $markdown += New-MDList -Style Unordered -Lines (@(
     (Get-7zipVersion),
+    (Get-ApacheVersion),
```

Comment:
Please create a table as we do for an example PSQL - https://github.com/actions/virtual-environments/blob/releases/win19/20210110/images/win/Windows2019-Readme.md

It's difficult to understand the default path for apache and nginx, service names, etc.

Reply:
I have moved these to a new heading "Servers"  similar to Databases and added tables with Version, ServiceName and Path for both apache and nginx.

Reply:
Updated markdown to match the macOS PR with one more field `ServiceName`.
https://github.com/actions/virtual-environments/pull/2517/files#issue-559383062